service-front/module/Application/view/application/pages/identity_check_passed.twig

```diff
@@ -10,34 +11,33 @@
 
         <div class="govuk-form-group">
             <fieldset class="govuk-fieldset">
-                <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
-                    <h1 class="govuk-fieldset__heading">
-                        Did the donor have assistance from a friend or family member on this call?
-                    </h1>
-                </legend>
-                <div class="govuk-radios" data-module="govuk-radios">
-                    <div class="govuk-radios__item">
-                        <input
-                                class="govuk-radios__input"
-                                id="yes"
-                                name="assistance_on_call"
-                                type="radio"
-                                value="Yes"
-                        >
-                        <label class="govuk-label govuk-radios__label"
-                               for="yes">Yes</label>
-                    </div>
-                    <div class="govuk-radios__item">
-                        <input
-                                class="govuk-radios__input"
-                                id="no"
-                                name="assistance_on_call"
-                                type="radio"
-                                value="No"
-                        >
-                        <label class="govuk-label govuk-radios__label"
-                               for="no">No</label>
-                    </div>
+                <div class="govuk-form-group">
+                    <fieldset class="govuk-fieldset" aria-describedby="contact-hint">
+                        <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+                            <h1 class="govuk-fieldset__heading">
+                                Did the donor have assistance from a friend or family member on this call?
+                            </h1>
+                        </legend>
+                        <div class="govuk-radios__item">
+                            <input class="govuk-radios__input" id="assistance-1" name="assistance" type="radio" value="no" data-aria-controls="conditional-assistance-1">
+                            <label class="govuk-label govuk-radios__label" for="assistance-1"  aria-label="Assistance provided - no">
```

Comment:
We don't need to add labels here. It may confuse screenreader users because the text on the page won't match what is announced.
```suggestion
                            <label class="govuk-label govuk-radios__label" for="assistance-1">
```